app/forms/access_form.rb

```diff
@@ -1,15 +1,26 @@
 # frozen_string_literal: true
 
 # A form object for updating the rights on an item
+# @abstract
 class AccessForm
   extend ActiveModel::Naming
-
-  # @param [Cocina::Models::DRO]
-  def initialize(model, apo)
+  # @param [Cocina::Models::DRO, Cocina::Models::Collection] model
+  def initialize(model, default_rights: nil)
     @model = model
-    @apo = apo
+    @default_rights = default_rights || 'citation-only'
+  end
+
+  # @param [HashWithIndifferentAccess] params the values from the form
+  # @option params [String] :rights the rights representation from the form (must be one of the keys in Dor::RightsMetadataDS::RIGHTS_TYPE_CODES, or 'default')
+  def validate(params)
+    rights = params[:rights]
+    return false unless valid_rights_options.include?(rights)
```

Comment:
I find it a little surprising that the abstract base class is calling a method that only lives in the concrete child classes. I might expect to see that method also defined in this class but with a dumb return value or perhaps an exception raised.

Reply:
But then it would be impossible to have code coverage on that method.

Reply:
Because you don't instantiate abstract classes and each concrete class overrides that method.

Reply:
@jcoyne Sure, but it's not impossible in Ruby to instantiate an abstract class if we cared enough about code coverage thereof, right? No big deal here, though. I don't think we *need* a dummy implementation in the parent class, especially if we have a code comment here about the expectation that child classes implement this method. Cool with adding that?

Reply:
I added a comment, so resolving conversation.